crates/rpc/src/types/syncing.rs

```diff
@@ -23,15 +22,36 @@ impl std::fmt::Display for Syncing {
     }
 }
 
+impl crate::dto::serialize::SerializeForVersion for Syncing {
+    fn serialize(
+        &self,
+        serializer: crate::dto::serialize::Serializer,
+    ) -> Result<crate::dto::serialize::Ok, crate::dto::serialize::Error> {
+        match self {
+            Syncing::False(b) => serializer.serialize_bool(*b),
```

Comment:
According to the spec the only possible value here is false, so we could just as well remove the `bool` from our `Syncing` enum now that we can properly serialize that variant into a `false` in JSON.

Reply:
Alright!